spec/features/user_index_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+RSpec.feature 'User index page', type: :feature do
+  before do
+    @user1 = User.create(name: 'Marco',
+                         photo: 'https://image.com/image.jpg', bio: 'Text for Bio', posts_counter: 0)
+
+    3.times do |i|
+      Post.create(title: "Post #{i + 1}", author: @user1, comments_counter: 0, likes_counter: 0)
+    end
+
+    @user2 = User.create(name: 'Tumaini',
+                         photo: 'https://image.com/image.jpg', bio: 'Text for Bio', posts_counter: 0)
+  end
+
+  scenario 'I can see the username of all other users' do
+    visit users_path
+
+    expect(page).to have_content('Marco')
+    expect(page).to have_content('Tumaini')
+  end
+
+  scenario 'I can see the profile picture for each user' do
+    visit users_path
+
+    expect(page).to have_css("img[src*='#{@user1.photo}']")
+    expect(page).to have_css("img[src*='#{@user2.photo}']")
```

Comment:
- [ ] Impressive with organizing your tests Team, But please consider using two different profile pictures for the two users. We need to ensure we can see each user's profile picture and for that, we need to use different pictures.

Reply:
Thanks